internal/testutil/util.go

```diff
@@ -43,7 +44,8 @@ import (
 
 	"github.com/fluxcd/pkg/gittestserver"
 
-	"github.com/fluxcd/image-automation-controller/pkg/update"
```

Comment:
Removing cyclic dependency now that a test from `pkg/update` imports `internal/testutil`